Typeform/quill
# fix(BubbleTooltip): Fix positioning for multiple line selection

## What does this PR do?
This PR extends the tooltip position correction for multiple line selections added here eb4a1cd , adding the same calculation for any optimize event related recalculations.

## How to reproduce?
1. Configure the editor with bubble theme
2. Type a 3 line text
3. Select everything
4. The toolbar tooltip appears in the correct position (thanks to previous fix)
5. Click the bold format or any format
6. The position of the tooltip is recalculated incorrectly

## Related issue
Improves the fix for this issue: #706
